kwanyong/app.js

```diff
@@ -0,0 +1,53 @@
+require("dotenv").config();     
+
+const express = require('express');
+const logger = require('morgan');
+const cors = require('cors');
+const { DataSource } = require('typeorm');
+
+
+const appDataSource = new DataSource({
+    type: process.env.DB_CONNECTION,
+    host: process.env.DB_HOST,
+    port: process.env.DB_PORT,
+    username: process.env.DB_USERNAME,
+    password: process.env.DB_PASSWORD,
+    database: process.env.DB_DATABASE
+})
+
+appDataSource.initialize()
+    .then(() => {
+        console.log("Data Source has been initialized!")
+    })
+
+const app = express()
+ 
+app.use(cors());                  
+app.use(logger('combined'));
+app.use(express.json());
+
+const port = process.env.PORT
+
+app.get('/ping', function (req, res, next) {   
+  res.json({message: 'userCreated'})
+})
+
+app.post('/users', async (req, res) => {
+	const { name, email, profileImage, password} = req.body
+    
+	await appDataSource.query(
+		`INSERT INTO users(
+		    name,
+		    email,
+		    profile_image,
+        password
+		) VALUES (?, ?, ?, ?);
+		`,
+		[ name, email, profileImage, password ]
+	); 
+     res.status(201).json({ message : "successfully created" });
+	})
```

Comment:
- 코드의 가독성을 높일 수 있는 다양한 요인이 있습니다.
해당 코드 블록을 보면 tab-size가 VSCode 상에서 다르게 적용되어 들여쓰기가 같은 파일 내부에서도 통일되지 않은 것 같습니다.

- 엔드포인트가 현재 `/users`로 설정되어 있습니다.
만약 로그인 기능이 추가될 때의 엔드포인트와 겹치지 않도록 회원가입을 가리키는 `/signup` 도 추가해보면 어떨까요?

```suggestion
app.post('/users/signup', async (req, res) => {
  const { name, email, profileImage, password} = req.body
    
  await appDataSource.query(
    `INSERT INTO users(
        name,
        email,
        profile_image,
        password
      ) VALUES (?, ?, ?, ?);
      `,
	[ name, email, profileImage, password ]
      ); 
     res.status(201).json({ message : "successfully created" });
})
```